src/bundle/Resources/views/fieldtypes/edit/ezimageasset.html.twig

```diff
@@ -0,0 +1,126 @@
+{% trans_default_domain 'ezrepoforms_content' %}
+
+{% use '@ezdesign/fieldtypes/edit/binary_base.html.twig' %}
+
+{%- block ezplatform_fieldtype_ezimageasset_row -%}
+    {% set preview_block_name = 'ezimageasset_preview' %}
+    {% set attr = attr|merge({'accept': 'image/*'}) %}
+    {% set file_is_empty = not form.parent.vars.value.value.destinationContentId %}
+    {% set max_file_size = form.vars.max_file_size %}
+    {% set language_code = form.parent.vars.languageCode %}
+    {% set attr = attr|merge({
+        'data-max-file-size': max_file_size,
+        'data-language-code': language_code
+    }) %}
+    {% set wrapper_attr = wrapper_attr|default({})|merge({'class': (wrapper_attr.class|default('') ~ ' ez-field-edit--with-preview')|trim}) %}
+    {% set preview_attr = preview_attr|default({})|merge({'class': (preview_attr.class|default('') ~ ' ez-field-edit__preview')|trim}) %}
+    {% set widget_wrapper_attr = widget_wrapper_attr|default({})|merge({'class': (widget_wrapper_attr.class|default('') ~ ' ez-field-edit__data')|trim}) %}
+
+    {% set has_error = false %}
+    {% for error in form.parent.parent.parent.vars.errors %}
+        {% if error.origin == form.vars.errors.form %}
+            {% set has_error = true %}
+        {% endif %}
+    {% endfor %}
+
+    {% if file_is_empty or has_error %}
+        {% set preview_attr = preview_attr|default({})|merge({'hidden': 'hidden'}) %}
+    {% else %}
+        {% set widget_wrapper_attr = widget_wrapper_attr|merge({'hidden': 'hidden'}) %}
+    {% endif %}
+
+    {% set widget_container_block = block('ezplatform_fieldtype_ezimageasset_widget_container') %}
+
+    {{ block('form_row_fieldtype') }}
```

Comment:
You are duplicating a lot of variables from block `binary_base_row` (defined in `binary_base.html.twig`). Can't you override only some of the variables (it might require to do some changes to `binary_base_row` to make it possible) and display `binary_base_row`?